html.js

```diff
@@ -46,6 +46,9 @@ const Html = ({ body }) => {
         <meta itemProp="image" content="/banner.png" />
 
         <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" />
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css" />
```

Comment:
What about importing the styles directly from the package? I don't like having two additional requests when we could bundle this with the rest of our `css`